scalableminds/webknossos
# Display rockefeller logo for rockefeller datasets

This is a band-aid fix. In the long run, we want to have configurable logo options per organization/dataset.

### URL of deployed dev instance (used for testing):
- https://rockefellerbandaid.webknossos.xyz

### Steps to test:
- open (view mode) a normal dataset and ensure the mpi logo is shown
- upload a dataset with "rockefeller" in it's name (don't rename an existing dataset) and ensure the rockefeller logo is shown (mpi logo should not be shown during "load time" <-- that's why I introduced the dummy dataset attribute)

------
